service.nextup.notification/service.py

```diff
@@ -34,16 +37,32 @@ def logMsg(self, msg, lvl=1):
     def ServiceEntryPoint(self):
         player = Player()
         monitor = xbmc.Monitor()
-
+        
         lastFile = None
         lastUnwatchedFile = None
+
+        ga = GoogleAnalytics()
```

Comment:
please remove it, we dont want add-ons with user tracking, esp. when it is opt-out.